lib/rucio/daemons/hermes/hermes.py

```diff
@@ -240,6 +240,45 @@ def aggregate_to_influx(
     return 204
 
 
+def build_message_dict(
+        bulk: int,
+        worker_number: int,
+        total_workers: int,
+        message_dict: dict[str, list[dict[str, Any]]],
+        logger: "LoggerFunction",
+        service: str = "",
```

Comment:
```suggestion
        service: Optional[str] = None,
```

I think this would be more consistent with the usage of `retrieve_messages`, which expects an `Optional[str]` too.